pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -295,6 +297,14 @@ type CanaryStrategy struct {
 	AbortScaleDownDelaySeconds *int32 `json:"abortScaleDownDelaySeconds,omitempty" protobuf:"varint,13,opt,name=abortScaleDownDelaySeconds"`
 }
 
+// Analysis configuration for the analysis runs to retain
+type Analysis struct {
```

Comment:
Nit: The type name of `Analysis` is a little too broad. I think it'll be better rename to `AnalysisStrategy`, since these will hold options on how to do analysis.

NOTE: I'm only talking about the golang type name and not the json field name.

Reply:
Renamed